### ISSUE TITLE
Static Page Edit not Saved

### ISSUE THREAD

**Issue by KamalRanaweera** [arcualberta/TAPoR#79]
The save button in the edit view of the About page does not work,

**Comment by lagoan** [comment]
It does work on my test. @KamalRanaweera , could you provide more information on how you did your test?

**Comment by KamalRanaweera** [comment]
I logged into http://tapor-test.artsrn.ualberta.ca/ with my CCID account, then clicked on the About link in the top navigation bar and then clicked on the Edit button. This brings me into http://tapor-test.artsrn.ualberta.ca/pages/edit/about_tapor page as expected. I also see the Save and Cancel buttons at the top. I can edit the text but I cannot save. The save button does not work. I tried both on Chrome and Firefox, neither worked.

I also noted that if I go to http://tapor-test.artsrn.ualberta.ca/pages/edit/about_tapor page without login in, it still gives me the edit page. Access to these Edit pages should only be available to admin users.

Thanks
